swauth/authtypes.py

```diff
@@ -100,4 +110,56 @@ def match(self, key, creds):
         :param creds: User's stored credentials
         :returns: True if the supplied key is valid, False otherwise
         """
-        return self.encode(key) == creds
+
+        [type, rest] = creds.split(':')
```

Comment:
My Python's a little rusty, so this is just a curiosity question, but why the brackets? Best I can tell they're optional or could even be parenthesis and I'm wondering if there is some subtle difference.